### ISSUE TITLE
Feature: List of users currently viewing the shoutbox

### ISSUE THREAD

**Issue by Radiergummi** [Schamper/nodebb-plugin-shoutbox#75]
I use the shoutbox as a chatroom on my board. The users request this all the time (as seen on topics).

**Comment by Schamper** [comment]
In the current version this is only displayed on the `/shoutbox` route. The rewrite I'm working on supports options though, so it'll be possible to toggle it even for the widget.

**Comment by fgallese** [comment]
@Schamper I can not see the list of users at /shoutbox, is is still supported on the current version ?

**Comment by FaizanZahid** [comment]
@Schamper  are you still working on this?

**Comment by Schamper** [comment]
@FaizanZahid not really, feel free to submit a PR and I'll merge it and update it on npm.

**Comment by arranka** [comment]
Is there any way to make this improvement?

**Comment by DroidBV8** [comment]
Great feature request
